src/app/(proper_react)/redesign/(authenticated)/user/dashboard/DashboardTopBanner.tsx

```diff
@@ -175,6 +176,33 @@ export const DashboardTopBanner = (props: DashboardTopBannerProps) => {
       ),
       learnMore: null,
     },
+    YourDataIsProtectedAllFixedContent: {
+      headline: l10n.getString(
+        "dashboard-top-banner-your-data-is-protected-title"
+      ),
+      description: l10n.getString(
+        "dashboard-top-banner-your-data-is-protected-all-fixed-description",
+        {
+          starting_exposure_total_num: totalExposures,
+        }
+      ),
+      cta: (
+        <Button
+          // Ignored for test coverage; to be implemented:
+          /* c8 ignore next 3 */
```

Comment:
This looks like it's implemented now, could it be tested already? I think the test should verify that the resolved breaches should be visible after clicking the CTA, and not before it?

```suggestion
```